aiutils/vis/image_io.py

```diff
@@ -1,54 +1,55 @@
-from termcolor import colored
 from PIL import Image
 import numpy as np
 import pdb
 
-
-def raise_exception(e, err_str):
-    pkg_err_str = 'Exception raised in package pyAIUtil: \n{}'.format(err_str)
-    print colored(pkg_err_str, 'red')
-    raise
-
-
-def check_assertion(cond, err_str):
-    pkg_err_str = '\nAssertion failed in package pyAIUtil: \n{}'.format(
-        err_str)
-    assert (cond), colored(pkg_err_str, 'blue')
+import aiutils.devtools.err_msg as err_msg
 
 
 def imread(filename):
+    """
+    Matlab like function for reading an image file.
+
+    Returns:
+    np_im (numpy.ndarray): h x w x 3 ndarray for color images and 
+        h x w for grayscale images
+    """
     try:
         im = Image.open(filename)
-
     except IOError as e:
         err_str = 'imread could not open file {}'.format(filename)
-        raise_exception(e, err_str)
+        err_msg.raise_exception(e, err_str)
 
     assert_cond = im.mode == 'RGB' or im.mode == 'L'
     err_str = \
         "imread only supports 'RGB' and 'L' modes, found '{}'".format(im.mode)
-    check_assertion(assert_cond, err_str)
+    err_msg.check_assertion(assert_cond, err_str)
 
     im_seq = np.array(im.getdata(), np.uint8)
```

Comment:
Does this have a different behavior than the simpler: `im_seq = np.array(im)`?

Also I think calling the variable `im_array` would be clearer.